sample-node-config/aws/README.md

```diff
@@ -0,0 +1,234 @@
+    ```
+    > `cardano.addr` will be generated at the root of the `hydra-poc` project.
+
+5. move under `credentials` folder:
+    - the party members verification keys: `arnaud-cardano.vk` and `arnaud-hydra.vk`.
+    - your personal hydra keys: `hydra-key.sk` and `hydra-key.vk`.
+    - your personal cardano keys: `cardano-key.sk` and `cardano-key.vk`.
+    - your personal cardano address: `cardano.addr`.
+
+    ```sh
+    $ mv *.vk sample-node-config/aws/credentials/
+    $ mv *.sk sample-node-config/aws/credentials/
+    $ mv cardano.addr sample-node-config/aws/credentials/
+    ```
+
+### Building ***env***
+This should only be done once, when starting afresh.
+This is a folder you must create yourself to store your personal aws credentials.
+
+go to ***EC2 Dashboard*** > ***Network & Security*** > ***Key Pairs*** and create one (for this template, the key pair is ***personal*** as mentioned).
+Then move it under `env` folder.
+
+> Default values are good enough (type: RSA & OpenSSH format: *.pem).
+
+> Creating a new pair will automatically generate a file and download it (one time action).
+
+## Initialise Terraform
+This should only be done once, when starting afresh.
+At the root of `aws` project execute:
+```sh
+$ terraform init
+```
+
+### Alter terraform variables
+
+Alter the file variables.tf found in `sample-node-config/aws/` to reflect your key name, github account etc.
+
+### Deploying the VM
+Then create a deployment plan and apply it:
+```sh
+$ terraform plan -out vm.plan
+```
+
+#### Starting VM
+
+```sh
+$ terraform apply vm.plan
+... <takes some time - 3m aprox>
+
+Apply complete! Resources: 18 added, 0 changed, 0 destroyed.
+
+Outputs:
+
+instance_ip = "ec2-13-38-62-128.eu-west-3.compute.amazonaws.com"
+```
+
+> execute `terraform destroy` to take it down
+
+# Using the Hydra Node
+
+Note: From now on we are assuming that the following commands will be executed inside of `sample-node-config/aws`
+
+``` sh
+$ cd sample-node-config/aws
+```
+
+## Login to the VM
+
+One should be able to log into the VM as user `ubuntu`.
+
+Note: Please make sure your `env/personal.pem` file has correct permissions (`chmod 400 env/personal.pem`)
+
+To login to the VM:
+
+```
+export AWS_PROFILE=<PERSONAL_PROFILE>
+
+$ scripts/login.sh
+```
+
+> useful tmux commands:
+- ctrl B + D: detach session
+- tmux a: attach the detached-session
+
+## Opening the Head
+Before you open a head you must have funds on the preview network and then
+prepare the funds to be marked as fuel.
+
+To get some funds from the preview faucet to your address, you can either:
+    + claim them from site:
+        https://faucet.preview.world.dev.cardano.org/basic-faucet
+    + or request them via http: 
+        ```sh
+        $ curl -X POST -s "https://faucet.preview.world.dev.cardano.org/send-money/$(cat credentials/cardano.addr)?api_key="
+        ```
+    
+        > to request via http you must first obtain your api_key.
```

Comment:
I would greatly appreciate a link to some doc on how to get an api key for faucet

Reply:
Not needed. There is https://faucet.preview.world.dev.cardano.org/basic-faucet